packages/components/date-input/src/date-input-group.tsx

```diff
@@ -7,7 +7,7 @@ import {dataAttr} from "@heroui/shared-utils";
 
 export interface DateInputGroupProps extends ValidationResult, HelpTextProps {
   children?: ReactElement | ReactElement[];
-  shouldLabelBeOutside?: boolean;
+  shouldLabelBeOutside?: false | "outside" | "outside-left";
```

Comment:
i don't think it's a good idea to include `outside` and `outside-left`. If you need to determine if it is outside or outside-left, why not using `labelPlacement` to check?